src/main/java/io/phasetwo/keycloak/themes/theme/FreeMarkerAndMustacheEmailTemplateProvider.java

```diff
@@ -56,6 +56,14 @@ private EmailTemplate processMustacheTemplate(
       Theme theme = getTheme();
       Locale locale = session.getContext().resolveLocale(user);
       attributes.put("locale", locale);
+
+      // Expose locale_xy flags
+      realm.getSupportedLocalesStream().forEach(loc -> {
+              String key = String.format("locale_%s", loc);
+              Boolean ok = locale.getLanguage().equals(new Locale(loc).getLanguage());
```

Comment:
This may be unnecessary. `java.util.Locale` overrides `equals()` so can't you just use `locale.equals(loc)` here? Or are there cases in which the object is not equals, but the value of `getLanguage()` is?